qcelemental/util/importing.py

```diff
@@ -23,7 +23,10 @@ def which_import(module: str, *, return_bool: bool = False, raise_error: bool =
     """
     import importlib
 
-    module_spec = importlib.util.find_spec(module, package=package)
+    try:
+        module_spec = importlib.util.find_spec(module, package=package)
+    except ModuleNotFoundError:
+        module_spec = None
```

Comment:
(from above) I think the try/catch can be avoided

Reply:
It turned out that the following uses of `importlib.util.find_spec` produce the same result:
```python
import importlib

importlib.util.find_spec("pint.util")

importlib.util.find_spec(".util", package="pint")
```
Both will produce a `ModuleNotFoundError`, which must be caught to handle for cases like `simtk.openmm`. I didn't properly notice this in #144 as doing

```
importlib.util.find_spec("util", package="pint")
```
does not throw an exception, but it also doesn't find the package, either (because we need the relative `'.util'`, not just `'util'`). It didn't register in my mind at the time that this didn't solve our problem.

The issue is that any attempt to use `find_spec` for a submodule always imports the module above first, and if it isn't present, we will get an exception. So, we have to catch it.

Reply:
Thanks, I think I understand better now. Ok with the try/except. Still think a `pint.evilutil` test would be worthwhile. And you may need to adapt the default msg https://github.com/MolSSI/QCElemental/pull/150/files#diff-8e6fced7103e77b67c18f7cd16df2298R34 for when both module and package are passed so that user not advised to install a non-top-level pkg.